### DISCUSSION TITLE
Is the project still alive?

### DISCUSSION THREAD

**Discussion by ElucGeek** [TwidereProject/TwidereX-Android#512] [General]
I used Twidere for years and start testing TwidereX in parallel. But now neither version got updates for months and none support thread propelry, which unfortunatelly are proliferating on Twitter.

Is the project still maintain? Should we expect updates and eventually thread support?

**Comment by Tlaster** [comment]
No, we just have a shortage of developers as we have several projects in development. We will try our best to update Twidere X frequently.
